Logging to shared log directory: /tmp/test_logs//1331863429
only running test 6
excepting tests: 15c
Starting client mft01: -o user_xattr,acl,flock mft01@tcp:/lustre /mnt/lustre2
Started clients mft01: 
mft01@tcp:/lustre on /mnt/lustre2 type lustre (rw,user_xattr,acl,flock)
debug=-1
subsystem_debug=0xffb7e3ff
debug_mb=2
mft01: Checking config lustre mounted on /mnt/lustre
Checking servers environments
Checking clients mft01 environments
Using TIMEOUT=20
disable quota as required
debug=-1
subsystem_debug=0xffb7e3ff
debug_mb=2
setting all flavor to null
already have total 8 null connections
.......

== replay-dual test 6: open1, open2, unlink |X| close1 [fail mds1] close2 == 04:04:00 (1331863440)
multiop /mnt/lustre2/a vo_c
TMPPIPE=/tmp/multiop_open_wait_pipe.16800
multiop /mnt/lustre/a vo_c
TMPPIPE=/tmp/multiop_open_wait_pipe.16800
Filesystem           1K-blocks      Used Available Use% Mounted on
mft01@tcp:/lustre       374928     49528    305272  14% /mnt/lustre
Failing mds1 on node mft01
Stopping /mnt/mds1 (opts:) on mft01
affected facets: mds1
Failover mds1 to mft01
04:04:14 (1331863454) waiting for mft01 network 900 secs ...
04:04:14 (1331863454) network interface is UP
Starting mds1: -o loop,user_xattr,acl  /tmp/lustre-mdt1 /mnt/mds1
debug=-1
subsystem_debug=0xffb7e3ff
debug_mb=2
Started lustre-MDT0000
Resetting fail_loc on all nodes...done.
PASS 6 (20s)
................== replay-dual replay-dual.sh test complete, duration 31 sec == 04:04:20 (1331863460)
Stopping clients: mft01 /mnt/lustre2 (opts:)
Stopping client mft01 /mnt/lustre2 opts:
